docs/BR.md

```diff
@@ -3031,7 +3031,6 @@ When encoding a `Name`, the CA SHALL ensure that:
   * Each `RelativeDistinguishedName` MUST contain exactly one `AttributeTypeAndValue`.
   * Each `RelativeDistinguishedName`, if present, is encoded within the `RDNSequence` in the order that it appears in [Section 7.1.4.2](#7142-subject-attribute-encoding).
     * For example, a `RelativeDistinguishedName` that contains a `countryName` `AttributeTypeAndValue` pair MUST be encoded within the `RDNSequence` before a `RelativeDistinguishedName` that contains a `stateOrProvinceName` `AttributeTypeAndValue`.
-  * Each `Name` MUST NOT contain more than one instance of a given `AttributeTypeAndValue` across all `RelativeDistinguishedName`s.
```

Comment:
I had understood the consensus to be that we'd add a carve-out to specifically add domainComponent as another field that allows multiple, as that was the only other field identified as being impacted by this change. Are there other fields which need to be present multiple times?

Reply:
I think it makes sense to add a carve out to allow multiple instances of `streetAddress` and `domainComponent` and prohibit all other attributes from appearing more than once.

Since OU is now prohibited, I can't think of any other attribute types that would need to appear more than once in an RDN.

Reply:
> We've been discussing multiple instances for a few years now, and streetAddress, OU, and domainComponent are the only three that have been identified as needing multiples. The situation is less clear in other ecosystems, but here on the TLS side, we have certificate transparency, so we're fairly confident there aren't other things hiding out here.
> 

As others have stated, the OU is currently not prohibited for CA Certificates.


> We're better off calling out explicitly the fields where it makes sense for repeats, instead of allowing arbitrary repeats for no reason, that may even have security implications (imagine, for example, your firewall and browser disagree on which of two needlessly repeated elements is the right one, and make security decisions that are inconsistent with each other).

I assume you are referring to multiple `commonName` subject attributes, because that's the only field that used for TLS which would still be a validated entry :)

Overall, I have no objection to call out explicit fields that allow multiple instances. As long as people are OK with adding the `domainComponent` field without any explicit validation rules, I can make the change.

Reply:
I believe the latest commits address the concerns raised about the `domainComponent` and `streetAddress` attributes. I made no changes to address the OU field in CA Certificates. If we want to explicitly allow that, please let me know.